### DISCUSSION TITLE
Version 24.05 uudet ominaisuudet -esittely to 7.11.2024 klo 13

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha#1458] [Koulutukset]
![versionvaihto](https://github.com/user-attachments/assets/266da0bd-b475-47a6-9796-6f03b2fd2b20)

Marraskuussa päivitetään Koha versioon 24.05 ja **torstaina 7.11.2024 klo 13** esitellään version uusia ominaisuuksia. Esittelijänä on Koha-Suomen pääkäyttäjä Anneli Österman. Esittely tallennetaan ja linkki tallenteeseen tulee tähän tiedotteeseen tilaisuuden jälkeen.

Tilaisuuteen ei tarvitse ilmoittautua ja siihen pääsee mukaan alla olevasta osallistumislinkistä.

[Osallistumislinkki](https://teams.microsoft.com/l/meetup-join/19%3ameeting_NDgzYzJiY2ItNDNhOC00MGViLTg4OTQtYjBhNmE5NjEzM2E0%40thread.v2/0?context=%7b%22Tid%22%3a%225cc89a67-fa29-4356-af5d-f436abc7c21b%22%2c%22Oid%22%3a%229fb701ba-ed03-4130-af66-87d3c323bb9a%22%7d)

**Comment by Thoriell** [comment]
Kuinka kauan tähän suunnilleen kannattaa varata aikaa?

**Reply by AnneliO** [reply]
Uskoisin tunnin riittävän hyvin. Edellisen versionvaihdon esittelyssä on mennyt reilu tunti ja siinä oli kahden version muutokset mukana. Tässä on vain yhden version verran. :)

**Comment by AnneliO** [comment]
Koulutuksen tallenne [Koha-Suomen Youtube-kanavalla](https://youtu.be/WU0UFnK2wgY)

Koulutuksen diasarja: [Uutta versiossa 24.05.pdf](https://github.com/user-attachments/files/17662407/Uutta_versiossa_2405.pdf)

**Reply by AnneliO** [reply]
Linkki tallenteeseen ja diasarja on nyt lisätty myös Koha-Suomen verkkosivuille Ohjeet -> Koulutukset -> [Kohan version 24.05 esittely](https://koha-suomi.fi/dokumentaatio/koulutukset/#kohan-2405--version-esittely)
